Techdocs documentation stuck at ""Documentation is accessed for the first time ..." for one specific set of docs

### 📜 Issue Labels

- [x] Please familiarize yourself with the issue labels used in this project: [LABELS.md](https://github.com/backstage/backstage/blob/master/LABELS.md)

### 🔎 Search Terms

```plain
Techdocs, build, stuck
```

### 🗃️ Project Area

TechDocs

### 🔗 External Integration

N/A

### 📝 Description & Context

We currently build our documentation on demand when users access the specific docs. This works for all entities but one. For this one specific entity the documentation page is stuck at "Documentation is accessed for the first time and is being prepared. The subsequent loads are much faster." The build logs only show "Waiting for logs...". In the backend logs I only see repeated "The docs building process is taking a little bit longer to process this entity. Please bear with us." even with `LOG_LEVEL=debug`.

When I build the docs locally with `npx @techdocs/cli serve` it works. Restarting or redeploying doesn't help. We run Backstage inside docker so each restart also means a clean filesystem (in the container).

### 👍 Expected Behavior

The documentation site is built and logs show output.

### 📦 Reproduction Repo

_No response_

### 🥾 Reproduction steps

1. Go to "Docs"
2. Open "Entity A" docs
3. Wait for the documentation site to be built.

Output of `yarn backstage-cli info` (omitted local section)
```plaintext
yarn backstage-cli info
OS:   Darwin 25.5.0 - darwin/arm64
node: v22.17.0
yarn: 4.5.1
cli:  0.36.0 (installed)
info module: 0.1.0
backstage:   1.49.3

Dependencies:
  @backstage-community/plugin-azure-devops                         0.6.0
  @backstage-community/plugin-azure-devops-backend                 0.7.2
  @backstage-community/plugin-azure-devops-common                  0.4.8
  @backstage-community/plugin-github-actions                       0.6.22
  @backstage-community/plugin-graphiql                             0.8.0
  @backstage-community/plugin-tech-radar                           0.7.10
  @backstage/app-defaults                                          1.7.6
  @backstage/backend-app-api                                       1.6.0
  @backstage/backend-common                                        0.24.1, 0.25.0
  @backstage/backend-defaults                                      0.16.0
  @backstage/backend-dev-utils                                     0.1.5, 0.1.7
  @backstage/backend-openapi-utils                                 0.6.7
  @backstage/backend-plugin-api                                    1.8.0, 0.8.1, 1.1.1
  @backstage/backend-tasks                                         0.6.1
  @backstage/backend-test-utils                                    1.11.1
  @backstage/catalog-client                                        1.14.0, 1.12.0, 1.9.1
  @backstage/catalog-model                                         1.7.7, 1.7.3, 1.7.5, 1.7.6
  @backstage/cli                                                   0.36.0
  @backstage/cli-common                                            0.2.1, 0.1.15, 0.2.0
  @backstage/cli-defaults                                          0.1.0
  @backstage/cli-module-actions                                    0.0.1
  @backstage/cli-module-auth                                       0.1.0
  @backstage/cli-module-build                                      0.1.0
  @backstage/cli-module-config                                     0.1.0
  @backstage/cli-module-github                                     0.1.0
  @backstage/cli-module-info                                       0.1.0
  @backstage/cli-module-lint                                       0.1.0
  @backstage/cli-module-maintenance                                0.1.0
  @backstage/cli-module-migrate                                    0.1.0
  @backstage/cli-module-new                                        0.1.0
  @backstage/cli-module-test-jest                                  0.1.0
  @backstage/cli-module-translations                               0.1.0
  @backstage/cli-node                                              0.3.1, 0.3.0
  @backstage/config                                                1.3.6, 1.3.2, 1.3.3, 1.3.7, 1.3.8
  @backstage/config-loader                                         1.10.9, 1.9.5
  @backstage/core-app-api                                          1.19.6, 1.15.4, 1.18.0, 1.19.0, 1.20.1
  @backstage/core-compat-api                                       0.5.9, 0.2.8, 0.3.5, 0.5.2
  @backstage/core-components                                       0.18.8, 0.13.9, 0.14.10, 0.15.1, 0.16.3, 0.18.1, 0.18.10
  @backstage/core-plugin-api                                       1.12.4, 1.10.3, 1.10.9, 1.11.0, 1.12.6
  @backstage/dev-utils                                             1.1.21
  @backstage/e2e-test-utils                                        0.1.2
  @backstage/errors                                                1.2.7, 1.3.0, 1.3.1
  @backstage/eslint-plugin                                         0.2.2
  @backstage/filter-predicates                                     0.1.1, 0.1.3
  @backstage/frontend-app-api                                      0.10.4, 0.13.0, 0.16.1, 0.16.3
  @backstage/frontend-defaults                                     0.5.0, 0.1.5, 0.3.1, 0.5.2
  @backstage/frontend-dev-utils                                    0.1.2
  @backstage/frontend-plugin-api                                   0.15.1, 0.12.0, 0.17.0, 0.7.0, 0.8.0, 0.9.4
  @backstage/frontend-test-utils                                   0.5.1, 0.2.5, 0.3.6
  @backstage/integration                                           2.0.0, 1.16.1, 1.18.0
  @backstage/integration-aws-node                                  0.1.15, 0.1.20
  @backstage/integration-react                                     1.2.16, 1.2.3, 1.2.10
  @backstage/module-federation-common                              0.1.2
  @backstage/plugin-api-docs                                       0.13.5
  @backstage/plugin-app                                            0.4.2
  @backstage/plugin-app-backend                                    0.5.12
  @backstage/plugin-app-node                                       0.1.43
  @backstage/plugin-app-react                                      0.2.1
  @backstage/plugin-auth                                           0.1.6
  @backstage/plugin-auth-backend                                   0.27.3
  @backstage/plugin-auth-backend-module-guest-provider             0.2.17
  @backstage/plugin-auth-backend-module-microsoft-provider         0.3.13
  @backstage/plugin-auth-node                                      0.6.14, 0.5.6
  @backstage/plugin-auth-react                                     0.1.25
  @backstage/plugin-catalog                                        2.0.1, 2.0.0
  @backstage/plugin-catalog-backend                                3.5.0
  @backstage/plugin-catalog-backend-module-azure                   0.3.15
  @backstage/plugin-catalog-backend-module-logs                    0.1.20
  @backstage/plugin-catalog-backend-module-msgraph                 0.9.1
  @backstage/plugin-catalog-backend-module-scaffolder-entity-model 0.2.18
  @backstage/plugin-catalog-common                                 1.1.8, 1.1.3, 1.1.5
  @backstage/plugin-catalog-graph                                  0.6.0
  @backstage/plugin-catalog-import                                 0.13.11
  @backstage/plugin-catalog-node                                   2.1.0, 1.15.1
  @backstage/plugin-catalog-react                                  2.1.1, 1.15.1, 1.21.1, 2.1.0
  @backstage/plugin-events-backend                                 0.6.0
  @backstage/plugin-events-node                                    0.4.20
  @backstage/plugin-mcp-actions-backend                            0.1.10
  @backstage/plugin-org                                            0.7.0
  @backstage/plugin-permission-backend                             0.7.10
  @backstage/plugin-permission-common                              0.9.7, 0.8.4, 0.9.1, 0.9.6, 0.9.8
  @backstage/plugin-permission-node                                0.10.11, 0.8.7
  @backstage/plugin-permission-react                               0.4.41, 0.4.30, 0.4.36
  @backstage/plugin-proxy-backend                                  0.6.11
  @backstage/plugin-proxy-node                                     0.1.13
  @backstage/plugin-scaffolder                                     1.36.1
  @backstage/plugin-scaffolder-backend                             3.2.0
  @backstage/plugin-scaffolder-common                              2.0.0
  @backstage/plugin-scaffolder-node                                0.13.0
  @backstage/plugin-scaffolder-node-test-utils                     0.3.9
  @backstage/plugin-scaffolder-react                               1.20.0
  @backstage/plugin-search                                         1.7.0
  @backstage/plugin-search-backend                                 2.1.0
  @backstage/plugin-search-backend-module-catalog                  0.3.13
  @backstage/plugin-search-backend-module-pg                       0.5.53
  @backstage/plugin-search-backend-module-techdocs                 0.4.12
  @backstage/plugin-search-backend-node                            1.4.2
  @backstage/plugin-search-common                                  1.2.23, 1.2.17, 1.2.19, 1.2.22
  @backstage/plugin-search-react                                   1.11.0
  @backstage/plugin-signals-react                                  0.0.20
  @backstage/plugin-techdocs                                       1.17.2
  @backstage/plugin-techdocs-backend                               2.1.6
  @backstage/plugin-techdocs-common                                0.1.1
  @backstage/plugin-techdocs-module-addons-contrib                 1.1.34
  @backstage/plugin-techdocs-node                                  1.14.4
  @backstage/plugin-techdocs-react                                 1.3.9, 1.2.13
  @backstage/plugin-user-settings                                  0.9.1
  @backstage/plugin-user-settings-common                           0.1.0
  @backstage/release-manifests                                     0.0.13
  @backstage/repo-tools                                            0.17.0
  @backstage/test-utils                                            1.7.16, 1.7.11, 1.7.4
  @backstage/theme                                                 0.7.2, 0.4.4, 0.5.7, 0.6.3, 0.6.8, 0.7.3
  @backstage/types                                                 1.2.2, 1.2.1
  @backstage/ui                                                    0.13.2, 0.13.0, 0.13.1, 0.15.0
  @backstage/version-bridge                                        1.0.10, 1.0.11, 1.0.12
  backstage-plugin-techdocs-addon-mermaid                          0.10.0
```

### Have you read the Code of Conduct?

- [x] I have read the [Code of Conduct](https://github.com/backstage/backstage/blob/master/CODE_OF_CONDUCT.md)

### Are you willing to submit PR?

Yes, but I would like some more guidance

@mohamedaminehamdi sure. 

We've traced the problem down to a mismatch in Node versions between our build environment and the Docker image we build. Our build environment uses 22 and the container image used Node 24. We use buildkit bind mounts to bind yarn cache folders when building the image:
```docker
RUN --mount=type=cache,target=/home/node/.cache/yarn,sharing=locked,uid=1000,gid=1000 \
    yarn workspaces focus --all --production && rm -rf "$(yarn cache clean)"
```
I guess some Node 22 stuff got used in the Node 24 image causing the issue.

I've inspected a previous build log from the Docker build step, and it reported some issues building native dependencies during the link step with `node-gyp` for `sqlite3`, `cpu-features`, and `ssh-2`.
<details>
<summary> Below are the logs for the yarn install link step:</summary>

```plaintext
2026-06-10T05:01:21.3075639Z #16 37.79 ➤ YN0000: ┌ Link step
2026-06-10T05:05:47.6460807Z #16 304.3 ➤ YN0007: │ core-js@npm:3.33.0 must be built because it never has been before or the last one failed
2026-06-10T05:05:47.8169303Z #16 304.3 ➤ YN0007: │ sqlite3@npm:5.1.6 [aa597] must be built because it never has been before or the last one failed
2026-06-10T05:05:47.8170408Z #16 304.3 ➤ YN0007: │ isolated-vm@npm:6.1.2 must be built because it never has been before or the last one failed
2026-06-10T05:05:47.8171477Z #16 304.3 ➤ YN0007: │ protobufjs@npm:7.2.5 must be built because it never has been before or the last one failed
2026-06-10T05:05:47.8172538Z #16 304.3 ➤ YN0007: │ protobufjs@npm:7.4.0 must be built because it never has been before or the last one failed
2026-06-10T05:05:47.8173557Z #16 304.3 ➤ YN0007: │ cpu-features@npm:0.0.9 must be built because it never has been before or the last one failed
2026-06-10T05:05:47.8174481Z #16 304.3 ➤ YN0007: │ keytar@npm:7.9.0 must be built because it never has been before or the last one failed
2026-06-10T05:05:47.8175527Z #16 304.3 ➤ YN0007: │ core-js@npm:2.6.12 must be built because it never has been before or the last one failed
2026-06-10T05:05:49.0341896Z #16 305.7 ➤ YN0000: │ core-js@npm:3.33.0 STDOUT Thank you for using core-js ( https://github.com/zloirock/core-js ) for polyfilling JavaScript standard library!
2026-06-10T05:05:49.1854254Z #16 305.7 ➤ YN0000: │ core-js@npm:3.33.0 STDOUT 
2026-06-10T05:05:49.1856301Z #16 305.7 ➤ YN0000: │ core-js@npm:3.33.0 STDOUT The project needs your help! Please consider supporting core-js:
2026-06-10T05:05:49.1857250Z #16 305.7 ➤ YN0000: │ core-js@npm:3.33.0 STDOUT > https://opencollective.com/core-js 
2026-06-10T05:05:49.1858067Z #16 305.7 ➤ YN0000: │ core-js@npm:3.33.0 STDOUT > https://patreon.com/zloirock 
2026-06-10T05:05:49.1858902Z #16 305.7 ➤ YN0000: │ core-js@npm:3.33.0 STDOUT > https://boosty.to/zloirock 
2026-06-10T05:05:49.1859843Z #16 305.7 ➤ YN0000: │ core-js@npm:3.33.0 STDOUT > bitcoin: bc1qlea7544qtsmj2rayg0lthvza9fau63ux0fstcz 
2026-06-10T05:05:49.1860653Z #16 305.7 ➤ YN0000: │ core-js@npm:3.33.0 STDOUT 
2026-06-10T05:05:49.1861923Z #16 305.7 ➤ YN0000: │ core-js@npm:3.33.0 STDOUT I highly recommend reading this: https://github.com/zloirock/core-js/blob/master/docs/2023-02-14-so-whats-next.md 
2026-06-10T05:05:49.1862896Z #16 305.7 ➤ YN0000: │ core-js@npm:3.33.0 STDOUT 
2026-06-10T05:05:49.2530875Z #16 305.9 ➤ YN0000: │ core-js@npm:2.6.12 STDOUT Thank you for using core-js ( https://github.com/zloirock/core-js ) for polyfilling JavaScript standard library!
2026-06-10T05:05:49.3941814Z #16 305.9 ➤ YN0000: │ core-js@npm:2.6.12 STDOUT 
2026-06-10T05:05:49.3943581Z #16 305.9 ➤ YN0000: │ core-js@npm:2.6.12 STDOUT The project needs your help! Please consider supporting of core-js on Open Collective or Patreon: 
2026-06-10T05:05:49.3944556Z #16 305.9 ➤ YN0000: │ core-js@npm:2.6.12 STDOUT > https://opencollective.com/core-js 
2026-06-10T05:05:49.3945607Z #16 305.9 ➤ YN0000: │ core-js@npm:2.6.12 STDOUT > https://www.patreon.com/zloirock 
2026-06-10T05:05:49.3946329Z #16 305.9 ➤ YN0000: │ core-js@npm:2.6.12 STDOUT 
2026-06-10T05:05:49.3947242Z #16 305.9 ➤ YN0000: │ core-js@npm:2.6.12 STDOUT Also, the author of core-js ( https://github.com/zloirock ) is looking for a good job -)
2026-06-10T05:05:49.3948009Z #16 305.9 ➤ YN0000: │ core-js@npm:2.6.12 STDOUT 
2026-06-10T05:05:49.3948860Z #16 306.0 ➤ YN0000: │ sqlite3@npm:5.1.6 [aa597] STDERR node-pre-gyp info it worked if it ends with ok
2026-06-10T05:05:49.5577769Z #16 306.0 ➤ YN0000: │ sqlite3@npm:5.1.6 [aa597] STDERR node-pre-gyp info using node-pre-gyp@1.0.11
2026-06-10T05:05:49.5580177Z #16 306.0 ➤ YN0000: │ sqlite3@npm:5.1.6 [aa597] STDERR node-pre-gyp info using node@24.16.0 | linux | x64
2026-06-10T05:05:49.7166553Z #16 306.3 ➤ YN0000: │ sqlite3@npm:5.1.6 [aa597] STDERR node-pre-gyp info check checked for "/app/node_modules/sqlite3/lib/binding/napi-v6-linux-glibc-x64/node_sqlite3.node" (not found)
2026-06-10T05:05:49.9640883Z #16 306.4 ➤ YN0000: │ sqlite3@npm:5.1.6 [aa597] STDERR node-pre-gyp http GET https://github.com/TryGhost/node-sqlite3/releases/download/v5.1.6/napi-v6-linux-glibc-x64.tar.gz
2026-06-10T05:05:49.9641967Z #16 306.4 ➤ YN0000: │ keytar@npm:7.9.0 STDERR (node:85) [DEP0176] DeprecationWarning: fs.R_OK is deprecated, use fs.constants.R_OK instead
2026-06-10T05:05:49.9643484Z #16 306.4 ➤ YN0000: │ keytar@npm:7.9.0 STDERR (Use `node --trace-deprecation ...` to show where the warning was created)
2026-06-10T05:05:50.0717803Z #16 306.7 ➤ YN0000: │ sqlite3@npm:5.1.6 [aa597] STDERR node-pre-gyp info install unpacking napi-v6-linux-glibc-x64/node_sqlite3.node
2026-06-10T05:05:50.2319477Z #16 306.8 ➤ YN0000: │ sqlite3@npm:5.1.6 [aa597] STDERR node-pre-gyp info extracted file count: 1 
2026-06-10T05:05:50.2322380Z #16 306.8 ➤ YN0000: │ sqlite3@npm:5.1.6 [aa597] STDERR node-pre-gyp info ok 
2026-06-10T05:05:50.2323575Z #16 306.8 ➤ YN0000: │ sqlite3@npm:5.1.6 [aa597] STDOUT [sqlite3] Success: "/app/node_modules/sqlite3/lib/binding/napi-v6-linux-glibc-x64/node_sqlite3.node" is installed via remote
2026-06-10T05:05:50.2324668Z #16 306.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR gyp info it worked if it ends with ok
2026-06-10T05:05:50.4370014Z #16 306.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR gyp info using node-gyp@9.4.0
2026-06-10T05:05:50.4376105Z #16 306.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR gyp info using node@24.16.0 | linux | x64
2026-06-10T05:05:50.4377125Z #16 306.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR (node:123) [DEP0060] DeprecationWarning: The `util._extend` API is deprecated. Please use Object.assign() instead.
2026-06-10T05:05:50.4378107Z #16 306.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR (Use `node --trace-deprecation ...` to show where the warning was created)
2026-06-10T05:05:50.4452271Z #16 307.1 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR gyp info find Python using Python version 3.13.5 found at "/usr/bin/python3"
2026-06-10T05:05:50.5701862Z #16 307.2 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR gyp WARN EACCES current user ("node") does not have permission to access the dev dir "/home/node/.cache/node-gyp/24.16.0"
2026-06-10T05:05:50.5704489Z #16 307.2 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR gyp WARN EACCES attempting to reinstall using temporary dev dir "/tmp/.node-gyp"
2026-06-10T05:05:50.8082635Z #16 307.2 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR gyp http GET https://nodejs.org/download/release/v24.16.0/node-v24.16.0-headers.tar.gz
2026-06-10T05:05:50.8088521Z #16 307.3 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR gyp http 200 https://nodejs.org/download/release/v24.16.0/node-v24.16.0-headers.tar.gz
2026-06-10T05:05:52.3634379Z #16 309.0 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR gyp http GET https://nodejs.org/download/release/v24.16.0/SHASUMS256.txt
2026-06-10T05:05:52.5512521Z #16 309.0 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR gyp http 200 https://nodejs.org/download/release/v24.16.0/SHASUMS256.txt
2026-06-10T05:05:52.5521160Z #16 309.0 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR gyp info spawn /usr/bin/python3
2026-06-10T05:05:52.5522707Z #16 309.0 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR gyp info spawn args [
2026-06-10T05:05:52.5523880Z #16 309.0 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR gyp info spawn args   '/app/node_modules/node-gyp/gyp/gyp_main.py',
2026-06-10T05:05:52.5526331Z #16 309.0 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR gyp info spawn args   'binding.gyp',
2026-06-10T05:05:52.5527125Z #16 309.0 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR gyp info spawn args   '-f',
2026-06-10T05:05:52.5527970Z #16 309.0 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR gyp info spawn args   'make',
2026-06-10T05:05:52.5528879Z #16 309.0 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR gyp info spawn args   '-I',
2026-06-10T05:05:52.5529806Z #16 309.0 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR gyp info spawn args   '/app/node_modules/cpu-features/build/config.gypi',
2026-06-10T05:05:52.5534449Z #16 309.0 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR gyp info spawn args   '-I',
2026-06-10T05:05:52.5535515Z #16 309.0 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR gyp info spawn args   '/app/node_modules/node-gyp/addon.gypi',
2026-06-10T05:05:52.5536272Z #16 309.0 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR gyp info spawn args   '-I',
2026-06-10T05:05:52.5537131Z #16 309.0 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR gyp info spawn args   '/tmp/.node-gyp/24.16.0/include/node/common.gypi',
2026-06-10T05:05:52.5538048Z #16 309.0 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR gyp info spawn args   '-Dlibrary=shared_library',
2026-06-10T05:05:52.5538881Z #16 309.0 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR gyp info spawn args   '-Dvisibility=default',
2026-06-10T05:05:52.5539848Z #16 309.0 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR gyp info spawn args   '-Dnode_root_dir=/tmp/.node-gyp/24.16.0',
2026-06-10T05:05:52.5540871Z #16 309.0 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR gyp info spawn args   '-Dnode_gyp_dir=/app/node_modules/node-gyp',
2026-06-10T05:05:52.5541928Z #16 309.0 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR gyp info spawn args   '-Dnode_lib_file=/tmp/.node-gyp/24.16.0/<(target_arch)/node.lib',
2026-06-10T05:05:52.5542784Z #16 309.0 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR gyp info spawn args   '-Dmodule_root_dir=/app/node_modules/cpu-features',
2026-06-10T05:05:52.5543319Z #16 309.0 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR gyp info spawn args   '-Dnode_engine=v8',
2026-06-10T05:05:52.5543967Z #16 309.0 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR gyp info spawn args   '--depth=.',
2026-06-10T05:05:52.5544766Z #16 309.0 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR gyp info spawn args   '--no-parallel',
2026-06-10T05:05:52.5545834Z #16 309.0 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR gyp info spawn args   '--generator-output',
2026-06-10T05:05:52.5546739Z #16 309.0 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR gyp info spawn args   'build',
2026-06-10T05:05:52.5547597Z #16 309.0 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR gyp info spawn args   '-Goutput_dir=.'
2026-06-10T05:05:52.5548583Z #16 309.0 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR gyp info spawn args ]
2026-06-10T05:05:53.0686167Z #16 309.7 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR gyp info spawn make
2026-06-10T05:05:53.2502828Z #16 309.7 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR gyp info spawn args [ 'BUILDTYPE=Release', '-C', 'build' ]
2026-06-10T05:05:53.2504122Z #16 309.7 ➤ YN0000: │ cpu-features@npm:0.0.9 STDOUT make: Entering directory '/app/node_modules/cpu-features/build'
2026-06-10T05:05:53.2505399Z #16 309.7 ➤ YN0000: │ cpu-features@npm:0.0.9 STDOUT   CC(target) Release/obj.target/cpu_features/deps/cpu_features/src/impl_aarch64_linux_or_android.o
2026-06-10T05:05:53.8106303Z #16 310.4 ➤ YN0000: │ cpu-features@npm:0.0.9 STDOUT   CC(target) Release/obj.target/cpu_features/deps/cpu_features/src/impl_aarch64_macos_or_iphone.o
2026-06-10T05:05:53.9274136Z #16 310.6 ➤ YN0000: │ cpu-features@npm:0.0.9 STDOUT   CC(target) Release/obj.target/cpu_features/deps/cpu_features/src/impl_arm_linux_or_android.o
2026-06-10T05:05:54.0318109Z #16 310.6 ➤ YN0000: │ cpu-features@npm:0.0.9 STDOUT   CC(target) Release/obj.target/cpu_features/deps/cpu_features/src/impl_mips_linux_or_android.o
2026-06-10T05:05:54.0319281Z #16 310.6 ➤ YN0000: │ cpu-features@npm:0.0.9 STDOUT   CC(target) Release/obj.target/cpu_features/deps/cpu_features/src/impl_ppc_linux.o
2026-06-10T05:05:54.0320436Z #16 310.7 ➤ YN0000: │ cpu-features@npm:0.0.9 STDOUT   CC(target) Release/obj.target/cpu_features/deps/cpu_features/src/impl_x86_freebsd.o
2026-06-10T05:05:54.9377601Z #16 311.6 ➤ YN0000: │ cpu-features@npm:0.0.9 STDOUT   CC(target) Release/obj.target/cpu_features/deps/cpu_features/src/impl_x86_linux_or_android.o
2026-06-10T05:05:55.4297995Z #16 312.1 ➤ YN0000: │ cpu-features@npm:0.0.9 STDOUT   CC(target) Release/obj.target/cpu_features/deps/cpu_features/src/impl_x86_macos.o
2026-06-10T05:05:55.5591646Z #16 312.1 ➤ YN0000: │ cpu-features@npm:0.0.9 STDOUT   CC(target) Release/obj.target/cpu_features/deps/cpu_features/src/impl_x86_windows.o
2026-06-10T05:05:55.5595025Z #16 312.1 ➤ YN0000: │ cpu-features@npm:0.0.9 STDOUT   CC(target) Release/obj.target/cpu_features/deps/cpu_features/src/filesystem.o
2026-06-10T05:05:55.5596944Z #16 312.2 ➤ YN0000: │ cpu-features@npm:0.0.9 STDOUT   CC(target) Release/obj.target/cpu_features/deps/cpu_features/src/stack_line_reader.o
2026-06-10T05:05:55.7340798Z #16 312.3 ➤ YN0000: │ cpu-features@npm:0.0.9 STDOUT   CC(target) Release/obj.target/cpu_features/deps/cpu_features/src/string_view.o
2026-06-10T05:05:55.7342019Z #16 312.4 ➤ YN0000: │ cpu-features@npm:0.0.9 STDOUT rm -f Release/obj.target/deps/cpu_features/cpu_features.a Release/obj.target/deps/cpu_features/cpu_features.a.ar-file-list; mkdir -p `dirname Release/obj.target/deps/cpu_features/cpu_features.a`
2026-06-10T05:05:55.9390175Z #16 312.4 ➤ YN0000: │ cpu-features@npm:0.0.9 STDOUT ar crs Release/obj.target/deps/cpu_features/cpu_features.a @Release/obj.target/deps/cpu_features/cpu_features.a.ar-file-list
2026-06-10T05:05:55.9391797Z #16 312.4 ➤ YN0000: │ cpu-features@npm:0.0.9 STDOUT   COPY Release/cpu_features.a
2026-06-10T05:05:55.9392455Z #16 312.4 ➤ YN0000: │ cpu-features@npm:0.0.9 STDOUT   CXX(target) Release/obj.target/cpufeatures/src/binding.o
2026-06-10T05:05:57.1590683Z #16 313.8 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR In file included from ../src/binding.cc:3:
2026-06-10T05:05:57.1591770Z #16 313.8 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR ../../nan/nan.h:208:40: error: expected template-name before '<' token
2026-06-10T05:05:57.1592732Z #16 313.8 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR   208 |     public v8::CopyablePersistentTraits<T> {};
2026-06-10T05:05:57.1593565Z #16 313.8 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR       |                                        ^
2026-06-10T05:05:57.2598532Z #16 313.8 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR ../../nan/nan.h:208:40: error: expected '{' before '<' token
2026-06-10T05:05:57.2600905Z #16 313.8 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR ../../nan/nan.h: In function 'bool Nan::IdleNotification(int)':
2026-06-10T05:05:57.2602243Z #16 313.8 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR ../../nan/nan.h:688:39: error: 'class v8::Isolate' has no member named 'IdleNotificationDeadline'
2026-06-10T05:05:57.2603574Z #16 313.8 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR   688 |     return v8::Isolate::GetCurrent()->IdleNotificationDeadline(
2026-06-10T05:05:57.2604725Z #16 313.8 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR       |                                       ^~~~~~~~~~~~~~~~~~~~~~~~
2026-06-10T05:05:57.2606093Z #16 313.8 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR ../../nan/nan.h: In constructor 'Nan::Utf8String::Utf8String(v8::Local<v8::Value>)':
2026-06-10T05:05:57.2608063Z #16 313.8 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR ../../nan/nan.h:1154:36: warning: 'int v8::String::WriteUtf8(v8::Isolate*, char*, int, int*, int) const' is deprecated: Use WriteUtf8V2 instead. [-Wdeprecated-declarations]
2026-06-10T05:05:57.2609561Z #16 313.8 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR  1154 |         length_ = string->WriteUtf8(v8::Isolate::GetCurrent(), str_,
2026-06-10T05:05:57.2610809Z #16 313.8 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR       |                   ~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
2026-06-10T05:05:57.2611971Z #16 313.8 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR  1155 |                                     static_cast<int>(len), 0, flags);
2026-06-10T05:05:57.2613089Z #16 313.8 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR       |                                     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
2026-06-10T05:05:57.2614278Z #16 313.8 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR In file included from /tmp/.node-gyp/24.16.0/include/node/v8-object.h:12,
2026-06-10T05:05:57.2623420Z #16 313.8 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR                  from /tmp/.node-gyp/24.16.0/include/node/v8-array-buffer.h:14,
2026-06-10T05:05:57.2624996Z #16 313.8 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR                  from /tmp/.node-gyp/24.16.0/include/node/v8.h:24,
2026-06-10T05:05:57.2626113Z #16 313.8 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR                  from /tmp/.node-gyp/24.16.0/include/node/node.h:74,
2026-06-10T05:05:57.2629216Z #16 313.8 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR                  from ../src/binding.cc:1:
2026-06-10T05:05:57.2630195Z #16 313.8 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR /tmp/.node-gyp/24.16.0/include/node/v8-primitive.h:212:7: note: declared here
2026-06-10T05:05:57.2631181Z #16 313.8 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR   212 |   int WriteUtf8(Isolate* isolate, char* buffer, int length = -1,
2026-06-10T05:05:57.2632024Z #16 313.8 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR       |       ^~~~~~~~~
2026-06-10T05:05:57.2633061Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR ../../nan/nan.h: In function 'v8::Local<v8::Value> Nan::Encode(const void*, size_t, Encoding)':
2026-06-10T05:05:57.2634407Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR ../../nan/nan.h:2348:24: warning: 'v8::Local<v8::Value> node::Encode(v8::Isolate*, const uint16_t*, size_t)' is deprecated: Use TryEncode(...) instead [-Wdeprecated-declarations]
2026-06-10T05:05:57.2635758Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR  2348 |     return node::Encode(
2026-06-10T05:05:57.2636565Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR       |            ~~~~~~~~~~~~^
2026-06-10T05:05:57.2637373Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR  2349 |         isolate
2026-06-10T05:05:57.2638192Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR       |         ~~~~~~~         
2026-06-10T05:05:57.2639055Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR  2350 |       , reinterpret_cast<const uint16_t *>(buf)
2026-06-10T05:05:57.2639937Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR       |       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
2026-06-10T05:05:57.2641053Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR  2351 |       , len / 2);
2026-06-10T05:05:57.2641801Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR       |       ~~~~~~~~~~        
2026-06-10T05:05:57.2642643Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR /tmp/.node-gyp/24.16.0/include/node/node.h:1268:50: note: declared here
2026-06-10T05:05:57.2643648Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR  1268 |                 NODE_EXTERN v8::Local<v8::Value> Encode(v8::Isolate* isolate,
2026-06-10T05:05:57.2644600Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR       |                                                  ^~~~~~
2026-06-10T05:05:57.2645769Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR /tmp/.node-gyp/24.16.0/include/node/node.h:116:42: note: in definition of macro 'NODE_DEPRECATED'
2026-06-10T05:05:57.2647016Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR   116 |     __attribute__((deprecated(message))) declarator
2026-06-10T05:05:57.2647918Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR       |                                          ^~~~~~~~~~
2026-06-10T05:05:57.2649219Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR ../../nan/nan.h:2353:24: warning: 'v8::Local<v8::Value> node::Encode(v8::Isolate*, const char*, size_t, encoding)' is deprecated: Use TryEncode(...) instead [-Wdeprecated-declarations]
2026-06-10T05:05:57.2650324Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR  2353 |     return node::Encode(
2026-06-10T05:05:57.2651209Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR       |            ~~~~~~~~~~~~^
2026-06-10T05:05:57.2652091Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR  2354 |         isolate
2026-06-10T05:05:57.2652945Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR       |         ~~~~~~~         
2026-06-10T05:05:57.2653761Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR  2355 |       , reinterpret_cast<const char *>(buf)
2026-06-10T05:05:57.2654675Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR       |       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
2026-06-10T05:05:57.2655746Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR  2356 |       , len
2026-06-10T05:05:57.2656530Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR       |       ~~~~~             
2026-06-10T05:05:57.2657273Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR  2357 |       , node_enc);
2026-06-10T05:05:57.2657986Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR       |       ~~~~~~~~~~~       
2026-06-10T05:05:57.2658822Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR /tmp/.node-gyp/24.16.0/include/node/node.h:1260:38: note: declared here
2026-06-10T05:05:57.2659859Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR  1260 |     NODE_EXTERN v8::Local<v8::Value> Encode(v8::Isolate* isolate,
2026-06-10T05:05:57.2660767Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR       |                                      ^~~~~~
2026-06-10T05:05:57.2661793Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR /tmp/.node-gyp/24.16.0/include/node/node.h:116:42: note: in definition of macro 'NODE_DEPRECATED'
2026-06-10T05:05:57.2662797Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR   116 |     __attribute__((deprecated(message))) declarator
2026-06-10T05:05:57.2663675Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR       |                                          ^~~~~~~~~~
2026-06-10T05:05:57.2665330Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR ../../nan/nan.h: In function 'void Nan::SetAccessor(v8::Local<v8::ObjectTemplate>, v8::Local<v8::String>, GetterCallback, SetterCallback, v8::Local<v8::Value>, v8::AccessControl, v8::PropertyAttribute, imp::Sig)':
2026-06-10T05:05:57.2666718Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR ../../nan/nan.h:2548:8: error: 'class v8::ObjectTemplate' has no member named 'SetAccessor'
2026-06-10T05:05:57.2667890Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR  2548 |   tpl->SetAccessor(
2026-06-10T05:05:57.2668646Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR       |        ^~~~~~~~~~~
2026-06-10T05:05:57.2670162Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR ../../nan/nan.h: In function 'void Nan::SetAccessor(v8::Local<v8::ObjectTemplate>, v8::Local<v8::String>, GetterCallback, SetterCallback, v8::Local<v8::Value>, v8::AccessControl, v8::PropertyAttribute)':
2026-06-10T05:05:57.2671459Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR ../../nan/nan.h:2594:8: error: 'class v8::ObjectTemplate' has no member named 'SetAccessor'
2026-06-10T05:05:57.2672376Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR  2594 |   tpl->SetAccessor(
2026-06-10T05:05:57.2673195Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR       |        ^~~~~~~~~~~
2026-06-10T05:05:57.2674588Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR ../../nan/nan.h: In function 'bool Nan::SetAccessor(v8::Local<v8::Object>, v8::Local<v8::String>, GetterCallback, SetterCallback, v8::Local<v8::Value>, v8::AccessControl, v8::PropertyAttribute)':
2026-06-10T05:05:57.2676123Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR ../../nan/nan.h:2638:15: error: 'class v8::Object' has no member named 'SetAccessor'
2026-06-10T05:05:57.2677088Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR  2638 |   return obj->SetAccessor(
2026-06-10T05:05:57.2677947Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR       |               ^~~~~~~~~~~
2026-06-10T05:05:57.2679530Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR ../../nan/nan.h: In function 'void Nan::SetNamedPropertyHandler(v8::Local<v8::ObjectTemplate>, PropertyGetterCallback, PropertySetterCallback, PropertyQueryCallback, PropertyDeleterCallback, PropertyEnumeratorCallback, v8::Local<v8::Value>)':
2026-06-10T05:05:57.2681848Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR ../../nan/nan.h:2715:7: error: invalid conversion from 'Nan::imp::NativePropertyGetter' {aka 'void (*)(v8::Local<v8::Name>, const v8::PropertyCallbackInfo<v8::Value>&)'} to 'v8::NamedPropertyGetterCallback' {aka 'v8::Intercepted (*)(v8::Local<v8::Name>, const v8::PropertyCallbackInfo<v8::Value>&)'} [-fpermissive]
2026-06-10T05:05:57.2683455Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR  2715 |       getter_, setter_, query_, deleter_, enumerator_, obj));
2026-06-10T05:05:57.2684359Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR       |       ^~~~~~~
2026-06-10T05:05:57.2685340Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR       |       |
2026-06-10T05:05:57.2686523Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR       |       Nan::imp::NativePropertyGetter {aka void (*)(v8::Local<v8::Name>, const v8::PropertyCallbackInfo<v8::Value>&)}
2026-06-10T05:05:57.2687693Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR In file included from /tmp/.node-gyp/24.16.0/include/node/v8-function.h:15,
2026-06-10T05:05:57.2688712Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR                  from /tmp/.node-gyp/24.16.0/include/node/v8.h:33:
2026-06-10T05:05:57.2690807Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR /tmp/.node-gyp/24.16.0/include/node/v8-template.h:792:35: note:   initializing argument 1 of 'v8::NamedPropertyHandlerConfiguration::NamedPropertyHandlerConfiguration(v8::NamedPropertyGetterCallback, v8::NamedPropertySetterCallback, v8::NamedPropertyQueryCallback, v8::NamedPropertyDeleterCallback, v8::NamedPropertyEnumeratorCallback, v8::Local<v8::Value>, v8::PropertyHandlerFlags)'
2026-06-10T05:05:57.2692568Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR   792 |       NamedPropertyGetterCallback getter,
2026-06-10T05:05:57.2693510Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR       |       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~
2026-06-10T05:05:57.2695697Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR ../../nan/nan.h:2715:16: error: invalid conversion from 'Nan::imp::NativePropertySetter' {aka 'void (*)(v8::Local<v8::Name>, v8::Local<v8::Value>, const v8::PropertyCallbackInfo<v8::Value>&)'} to 'v8::NamedPropertySetterCallback' {aka 'v8::Intercepted (*)(v8::Local<v8::Name>, v8::Local<v8::Value>, const v8::PropertyCallbackInfo<void>&)'} [-fpermissive]
2026-06-10T05:05:57.2697660Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR  2715 |       getter_, setter_, query_, deleter_, enumerator_, obj));
2026-06-10T05:05:57.2698573Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR       |                ^~~~~~~
2026-06-10T05:05:57.2699400Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR       |                |
2026-06-10T05:05:57.2700673Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR       |                Nan::imp::NativePropertySetter {aka void (*)(v8::Local<v8::Name>, v8::Local<v8::Value>, const v8::PropertyCallbackInfo<v8::Value>&)}
2026-06-10T05:05:57.2703001Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR /tmp/.node-gyp/24.16.0/include/node/v8-template.h:793:35: note:   initializing argument 2 of 'v8::NamedPropertyHandlerConfiguration::NamedPropertyHandlerConfiguration(v8::NamedPropertyGetterCallback, v8::NamedPropertySetterCallback, v8::NamedPropertyQueryCallback, v8::NamedPropertyDeleterCallback, v8::NamedPropertyEnumeratorCallback, v8::Local<v8::Value>, v8::PropertyHandlerFlags)'
2026-06-10T05:05:57.2704727Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR   793 |       NamedPropertySetterCallback setter = nullptr,
2026-06-10T05:05:57.2705964Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR       |       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~
2026-06-10T05:05:57.2707645Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR ../../nan/nan.h:2715:25: error: invalid conversion from 'Nan::imp::NativePropertyQuery' {aka 'void (*)(v8::Local<v8::Name>, const v8::PropertyCallbackInfo<v8::Integer>&)'} to 'v8::NamedPropertyQueryCallback' {aka 'v8::Intercepted (*)(v8::Local<v8::Name>, const v8::PropertyCallbackInfo<v8::Integer>&)'} [-fpermissive]
2026-06-10T05:05:57.2709086Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR  2715 |       getter_, setter_, query_, deleter_, enumerator_, obj));
2026-06-10T05:05:57.2710022Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR       |                         ^~~~~~
2026-06-10T05:05:57.2710887Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR       |                         |
2026-06-10T05:05:57.2712043Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR       |                         Nan::imp::NativePropertyQuery {aka void (*)(v8::Local<v8::Name>, const v8::PropertyCallbackInfo<v8::Integer>&)}
2026-06-10T05:05:57.2714180Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR /tmp/.node-gyp/24.16.0/include/node/v8-template.h:794:34: note:   initializing argument 3 of 'v8::NamedPropertyHandlerConfiguration::NamedPropertyHandlerConfiguration(v8::NamedPropertyGetterCallback, v8::NamedPropertySetterCallback, v8::NamedPropertyQueryCallback, v8::NamedPropertyDeleterCallback, v8::NamedPropertyEnumeratorCallback, v8::Local<v8::Value>, v8::PropertyHandlerFlags)'
2026-06-10T05:05:57.2715932Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR   794 |       NamedPropertyQueryCallback query = nullptr,
2026-06-10T05:05:57.2716880Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR       |       ~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~
2026-06-10T05:05:57.2718394Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR ../../nan/nan.h:2715:33: error: invalid conversion from 'void (*)(v8::Local<v8::Name>, const v8::PropertyCallbackInfo<v8::Boolean>&)' to 'v8::NamedPropertyDeleterCallback' {aka 'v8::Intercepted (*)(v8::Local<v8::Name>, const v8::PropertyCallbackInfo<v8::Boolean>&)'} [-fpermissive]
2026-06-10T05:05:57.2719813Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR  2715 |       getter_, setter_, query_, deleter_, enumerator_, obj));
2026-06-10T05:05:57.2720773Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR       |                                 ^~~~~~~~
2026-06-10T05:05:57.2721670Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR       |                                 |
2026-06-10T05:05:57.2722811Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR       |                                 void (*)(v8::Local<v8::Name>, const v8::PropertyCallbackInfo<v8::Boolean>&)
2026-06-10T05:05:57.2725557Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR /tmp/.node-gyp/24.16.0/include/node/v8-template.h:795:36: note:   initializing argument 4 of 'v8::NamedPropertyHandlerConfiguration::NamedPropertyHandlerConfiguration(v8::NamedPropertyGetterCallback, v8::NamedPropertySetterCallback, v8::NamedPropertyQueryCallback, v8::NamedPropertyDeleterCallback, v8::NamedPropertyEnumeratorCallback, v8::Local<v8::Value>, v8::PropertyHandlerFlags)'
2026-06-10T05:05:57.2727142Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR   795 |       NamedPropertyDeleterCallback deleter = nullptr,
2026-06-10T05:05:57.2728071Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR       |       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~
2026-06-10T05:05:57.2729420Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR ../../nan/nan.h: In function 'void Nan::SetIndexedPropertyHandler(v8::Local<v8::ObjectTemplate>, IndexGetterCallback, IndexSetterCallback, IndexQueryCallback, IndexDeleterCallback, IndexEnumeratorCallback, v8::Local<v8::Value>)':
2026-06-10T05:05:57.2731487Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR ../../nan/nan.h:2785:7: error: invalid conversion from 'Nan::imp::NativeIndexGetter' {aka 'void (*)(unsigned int, const v8::PropertyCallbackInfo<v8::Value>&)'} to 'v8::IndexedPropertyGetterCallbackV2' {aka 'v8::Intercepted (*)(unsigned int, const v8::PropertyCallbackInfo<v8::Value>&)'} [-fpermissive]
2026-06-10T05:05:57.2732936Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR  2785 |       getter_, setter_, query_, deleter_, enumerator_, obj));
2026-06-10T05:05:57.2733827Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR       |       ^~~~~~~
2026-06-10T05:05:57.2734650Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR       |       |
2026-06-10T05:05:57.2797363Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR       |       Nan::imp::NativeIndexGetter {aka void (*)(unsigned int, const v8::PropertyCallbackInfo<v8::Value>&)}
2026-06-10T05:05:57.2799655Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR /tmp/.node-gyp/24.16.0/include/node/v8-template.h:871:39: note:   initializing argument 1 of 'v8::IndexedPropertyHandlerConfiguration::IndexedPropertyHandlerConfiguration(v8::IndexedPropertyGetterCallbackV2, v8::IndexedPropertySetterCallbackV2, v8::IndexedPropertyQueryCallbackV2, v8::IndexedPropertyDeleterCallbackV2, v8::IndexedPropertyEnumeratorCallback, v8::Local<v8::Value>, v8::PropertyHandlerFlags)'
2026-06-10T05:05:57.2801350Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR   871 |       IndexedPropertyGetterCallbackV2 getter = nullptr,
2026-06-10T05:05:57.2802417Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR       |       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~
2026-06-10T05:05:57.2804224Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR ../../nan/nan.h:2785:16: error: invalid conversion from 'Nan::imp::NativeIndexSetter' {aka 'void (*)(unsigned int, v8::Local<v8::Value>, const v8::PropertyCallbackInfo<v8::Value>&)'} to 'v8::IndexedPropertySetterCallbackV2' {aka 'v8::Intercepted (*)(unsigned int, v8::Local<v8::Value>, const v8::PropertyCallbackInfo<void>&)'} [-fpermissive]
2026-06-10T05:05:57.2825705Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR  2785 |       getter_, setter_, query_, deleter_, enumerator_, obj));
2026-06-10T05:05:57.2826952Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR       |                ^~~~~~~
2026-06-10T05:05:57.2827889Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR       |                |
2026-06-10T05:05:57.2829384Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR       |                Nan::imp::NativeIndexSetter {aka void (*)(unsigned int, v8::Local<v8::Value>, const v8::PropertyCallbackInfo<v8::Value>&)}
2026-06-10T05:05:57.2832146Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR /tmp/.node-gyp/24.16.0/include/node/v8-template.h:872:39: note:   initializing argument 2 of 'v8::IndexedPropertyHandlerConfiguration::IndexedPropertyHandlerConfiguration(v8::IndexedPropertyGetterCallbackV2, v8::IndexedPropertySetterCallbackV2, v8::IndexedPropertyQueryCallbackV2, v8::IndexedPropertyDeleterCallbackV2, v8::IndexedPropertyEnumeratorCallback, v8::Local<v8::Value>, v8::PropertyHandlerFlags)'
2026-06-10T05:05:57.2835117Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR   872 |       IndexedPropertySetterCallbackV2 setter = nullptr,
2026-06-10T05:05:57.2836222Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR       |       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~
2026-06-10T05:05:57.2838563Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR ../../nan/nan.h:2785:25: error: invalid conversion from 'Nan::imp::NativeIndexQuery' {aka 'void (*)(unsigned int, const v8::PropertyCallbackInfo<v8::Integer>&)'} to 'v8::IndexedPropertyQueryCallbackV2' {aka 'v8::Intercepted (*)(unsigned int, const v8::PropertyCallbackInfo<v8::Integer>&)'} [-fpermissive]
2026-06-10T05:05:57.2840740Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR  2785 |       getter_, setter_, query_, deleter_, enumerator_, obj));
2026-06-10T05:05:57.2841887Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR       |                         ^~~~~~
2026-06-10T05:05:57.2843043Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR       |                         |
2026-06-10T05:05:57.2844259Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR       |                         Nan::imp::NativeIndexQuery {aka void (*)(unsigned int, const v8::PropertyCallbackInfo<v8::Integer>&)}
2026-06-10T05:05:57.2847107Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR /tmp/.node-gyp/24.16.0/include/node/v8-template.h:873:38: note:   initializing argument 3 of 'v8::IndexedPropertyHandlerConfiguration::IndexedPropertyHandlerConfiguration(v8::IndexedPropertyGetterCallbackV2, v8::IndexedPropertySetterCallbackV2, v8::IndexedPropertyQueryCallbackV2, v8::IndexedPropertyDeleterCallbackV2, v8::IndexedPropertyEnumeratorCallback, v8::Local<v8::Value>, v8::PropertyHandlerFlags)'
2026-06-10T05:05:57.2849338Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR   873 |       IndexedPropertyQueryCallbackV2 query = nullptr,
2026-06-10T05:05:57.2850744Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR       |       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~
2026-06-10T05:05:57.2853014Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR ../../nan/nan.h:2785:33: error: invalid conversion from 'Nan::imp::NativeIndexDeleter' {aka 'void (*)(unsigned int, const v8::PropertyCallbackInfo<v8::Boolean>&)'} to 'v8::IndexedPropertyDeleterCallbackV2' {aka 'v8::Intercepted (*)(unsigned int, const v8::PropertyCallbackInfo<v8::Boolean>&)'} [-fpermissive]
2026-06-10T05:05:57.2855100Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR  2785 |       getter_, setter_, query_, deleter_, enumerator_, obj));
2026-06-10T05:05:57.2856134Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR       |                                 ^~~~~~~~
2026-06-10T05:05:57.2857262Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR       |                                 |
2026-06-10T05:05:57.2858413Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR       |                                 Nan::imp::NativeIndexDeleter {aka void (*)(unsigned int, const v8::PropertyCallbackInfo<v8::Boolean>&)}
2026-06-10T05:05:57.2860560Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR /tmp/.node-gyp/24.16.0/include/node/v8-template.h:874:40: note:   initializing argument 4 of 'v8::IndexedPropertyHandlerConfiguration::IndexedPropertyHandlerConfiguration(v8::IndexedPropertyGetterCallbackV2, v8::IndexedPropertySetterCallbackV2, v8::IndexedPropertyQueryCallbackV2, v8::IndexedPropertyDeleterCallbackV2, v8::IndexedPropertyEnumeratorCallback, v8::Local<v8::Value>, v8::PropertyHandlerFlags)'
2026-06-10T05:05:57.2862627Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR   874 |       IndexedPropertyDeleterCallbackV2 deleter = nullptr,
2026-06-10T05:05:57.2863607Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR       |       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~
2026-06-10T05:05:57.2864750Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR In file included from ../../nan/nan.h:2950:
2026-06-10T05:05:57.2866170Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR ../../nan/nan_scriptorigin.h: In constructor 'Nan::ScriptOrigin::ScriptOrigin(v8::Local<v8::Value>)':
2026-06-10T05:05:57.2867337Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR ../../nan/nan_scriptorigin.h:19:55: error: no matching function for call to 'v8::ScriptOrigin::ScriptOrigin(v8::Isolate*, v8::Local<v8::Value>&)'
2026-06-10T05:05:57.2868580Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR    19 |       v8::ScriptOrigin(v8::Isolate::GetCurrent(), name) {}
2026-06-10T05:05:57.2869152Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR       |                                                       ^
2026-06-10T05:05:57.2869694Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR In file included from /tmp/.node-gyp/24.16.0/include/node/v8-script.h:20,
2026-06-10T05:05:57.2870300Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR                  from /tmp/.node-gyp/24.16.0/include/node/v8-debug.h:10,
2026-06-10T05:05:57.2870871Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR                  from /tmp/.node-gyp/24.16.0/include/node/v8-isolate.h:19,
2026-06-10T05:05:57.2871435Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR                  from /tmp/.node-gyp/24.16.0/include/node/v8-snapshot.h:9,
2026-06-10T05:05:57.2872014Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR                  from /tmp/.node-gyp/24.16.0/include/node/v8-context.h:15,
2026-06-10T05:05:57.2872564Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR                  from /tmp/.node-gyp/24.16.0/include/node/v8.h:26:
2026-06-10T05:05:57.2873845Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR /tmp/.node-gyp/24.16.0/include/node/v8-message.h:65:13: note: candidate: 'v8::ScriptOrigin::ScriptOrigin(v8::Local<v8::Value>, int, int, bool, int, v8::Local<v8::Value>, bool, bool, bool, v8::Local<v8::Data>)'
2026-06-10T05:05:57.2874744Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR    65 |   V8_INLINE ScriptOrigin(Local<Value> resource_name,
2026-06-10T05:05:57.2875546Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR       |             ^~~~~~~~~~~~
2026-06-10T05:05:57.2876231Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR /tmp/.node-gyp/24.16.0/include/node/v8-message.h:65:39: note:   no known conversion for argument 1 from 'v8::Isolate*' to 'v8::Local<v8::Value>'
2026-06-10T05:05:57.2876973Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR    65 |   V8_INLINE ScriptOrigin(Local<Value> resource_name,
2026-06-10T05:05:57.2877505Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR       |                          ~~~~~~~~~~~~~^~~~~~~~~~~~~
2026-06-10T05:05:57.2878197Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR /tmp/.node-gyp/24.16.0/include/node/v8-message.h:63:17: note: candidate: 'constexpr v8::ScriptOrigin::ScriptOrigin(const v8::ScriptOrigin&)'
2026-06-10T05:05:57.2879064Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR    63 | class V8_EXPORT ScriptOrigin {
2026-06-10T05:05:57.2880009Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR       |                 ^~~~~~~~~~~~
2026-06-10T05:05:57.2881019Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR /tmp/.node-gyp/24.16.0/include/node/v8-message.h:63:17: note:   candidate expects 1 argument, 2 provided
2026-06-10T05:05:57.2905384Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR /tmp/.node-gyp/24.16.0/include/node/v8-message.h:63:17: note: candidate: 'constexpr v8::ScriptOrigin::ScriptOrigin(v8::ScriptOrigin&&)'
2026-06-10T05:05:57.2909032Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR /tmp/.node-gyp/24.16.0/include/node/v8-message.h:63:17: note:   candidate expects 1 argument, 2 provided
2026-06-10T05:05:57.2910305Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR ../../nan/nan_scriptorigin.h: In constructor 'Nan::ScriptOrigin::ScriptOrigin(v8::Local<v8::Value>, v8::Local<v8::Integer>)':
2026-06-10T05:05:57.2912053Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR ../../nan/nan_scriptorigin.h:25:52: error: no matching function for call to 'v8::ScriptOrigin::ScriptOrigin(v8::Isolate*, v8::Local<v8::Value>&, int)'
2026-06-10T05:05:57.4423304Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR    25 |                    , To<int32_t>(line).FromMaybe(0)) {}
2026-06-10T05:05:57.4430143Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR       |                                                    ^
2026-06-10T05:05:57.4431489Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR /tmp/.node-gyp/24.16.0/include/node/v8-message.h:65:13: note: candidate: 'v8::ScriptOrigin::ScriptOrigin(v8::Local<v8::Value>, int, int, bool, int, v8::Local<v8::Value>, bool, bool, bool, v8::Local<v8::Data>)'
2026-06-10T05:05:57.4432649Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR    65 |   V8_INLINE ScriptOrigin(Local<Value> resource_name,
2026-06-10T05:05:57.4433482Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR       |             ^~~~~~~~~~~~
2026-06-10T05:05:57.4434524Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR /tmp/.node-gyp/24.16.0/include/node/v8-message.h:65:39: note:   no known conversion for argument 1 from 'v8::Isolate*' to 'v8::Local<v8::Value>'
2026-06-10T05:05:57.4435785Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR    65 |   V8_INLINE ScriptOrigin(Local<Value> resource_name,
2026-06-10T05:05:57.4436644Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR       |                          ~~~~~~~~~~~~~^~~~~~~~~~~~~
2026-06-10T05:05:57.4437755Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR /tmp/.node-gyp/24.16.0/include/node/v8-message.h:63:17: note: candidate: 'constexpr v8::ScriptOrigin::ScriptOrigin(const v8::ScriptOrigin&)'
2026-06-10T05:05:57.4438761Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR    63 | class V8_EXPORT ScriptOrigin {
2026-06-10T05:05:57.4439544Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR       |                 ^~~~~~~~~~~~
2026-06-10T05:05:57.4440522Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR /tmp/.node-gyp/24.16.0/include/node/v8-message.h:63:17: note:   candidate expects 1 argument, 3 provided
2026-06-10T05:05:57.4441723Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR /tmp/.node-gyp/24.16.0/include/node/v8-message.h:63:17: note: candidate: 'constexpr v8::ScriptOrigin::ScriptOrigin(v8::ScriptOrigin&&)'
2026-06-10T05:05:57.4442868Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR /tmp/.node-gyp/24.16.0/include/node/v8-message.h:63:17: note:   candidate expects 1 argument, 3 provided
2026-06-10T05:05:57.4444110Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR ../../nan/nan_scriptorigin.h: In constructor 'Nan::ScriptOrigin::ScriptOrigin(v8::Local<v8::Value>, v8::Local<v8::Integer>, v8::Local<v8::Integer>)':
2026-06-10T05:05:57.4445754Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR ../../nan/nan_scriptorigin.h:33:54: error: no matching function for call to 'v8::ScriptOrigin::ScriptOrigin(v8::Isolate*, v8::Local<v8::Value>&, int, int)'
2026-06-10T05:05:57.4446816Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR    33 |                    , To<int32_t>(column).FromMaybe(0)) {}
2026-06-10T05:05:57.4447657Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR       |                                                      ^
2026-06-10T05:05:57.4448904Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR /tmp/.node-gyp/24.16.0/include/node/v8-message.h:65:13: note: candidate: 'v8::ScriptOrigin::ScriptOrigin(v8::Local<v8::Value>, int, int, bool, int, v8::Local<v8::Value>, bool, bool, bool, v8::Local<v8::Data>)'
2026-06-10T05:05:57.4450032Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR    65 |   V8_INLINE ScriptOrigin(Local<Value> resource_name,
2026-06-10T05:05:57.4450837Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR       |             ^~~~~~~~~~~~
2026-06-10T05:05:57.4451939Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR /tmp/.node-gyp/24.16.0/include/node/v8-message.h:65:39: note:   no known conversion for argument 1 from 'v8::Isolate*' to 'v8::Local<v8::Value>'
2026-06-10T05:05:57.4453580Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR    65 |   V8_INLINE ScriptOrigin(Local<Value> resource_name,
2026-06-10T05:05:57.4454588Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR       |                          ~~~~~~~~~~~~~^~~~~~~~~~~~~
2026-06-10T05:05:57.4456161Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR /tmp/.node-gyp/24.16.0/include/node/v8-message.h:63:17: note: candidate: 'constexpr v8::ScriptOrigin::ScriptOrigin(const v8::ScriptOrigin&)'
2026-06-10T05:05:57.4457142Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR    63 | class V8_EXPORT ScriptOrigin {
2026-06-10T05:05:57.4457887Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR       |                 ^~~~~~~~~~~~
2026-06-10T05:05:57.4458808Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR /tmp/.node-gyp/24.16.0/include/node/v8-message.h:63:17: note:   candidate expects 1 argument, 4 provided
2026-06-10T05:05:57.4459989Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR /tmp/.node-gyp/24.16.0/include/node/v8-message.h:63:17: note: candidate: 'constexpr v8::ScriptOrigin::ScriptOrigin(v8::ScriptOrigin&&)'
2026-06-10T05:05:57.4461120Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR /tmp/.node-gyp/24.16.0/include/node/v8-message.h:63:17: note:   candidate expects 1 argument, 4 provided
2026-06-10T05:05:57.4462030Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR ../src/binding.cc: At global scope:
2026-06-10T05:05:57.4463738Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR /tmp/.node-gyp/24.16.0/include/node/node.h:1358:7: warning: cast between incompatible function types from 'void (*)(Nan::ADDON_REGISTER_FUNCTION_ARGS_TYPE)' {aka 'void (*)(v8::Local<v8::Object>)'} to 'node::addon_register_func' {aka 'void (*)(v8::Local<v8::Object>, v8::Local<v8::Value>, void*)'} [-Wcast-function-type]
2026-06-10T05:05:57.4465351Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR  1358 |       (node::addon_register_func) (regfunc),                          \
2026-06-10T05:05:57.4466250Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR       |       ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
2026-06-10T05:05:57.4467338Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR /tmp/.node-gyp/24.16.0/include/node/node.h:1392:3: note: in expansion of macro 'NODE_MODULE_X'
2026-06-10T05:05:57.4468419Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR  1392 |   NODE_MODULE_X(modname, regfunc, NULL, 0)  // NOLINT (readability/null_usage)
2026-06-10T05:05:57.4469302Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR       |   ^~~~~~~~~~~~~
2026-06-10T05:05:57.4470185Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR ../src/binding.cc:151:1: note: in expansion of macro 'NODE_MODULE'
2026-06-10T05:05:57.4471084Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR   151 | NODE_MODULE(cpufeatures, init)
2026-06-10T05:05:57.4471861Z #16 313.9 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR       | ^~~~~~~~~~~
2026-06-10T05:05:57.5059260Z #16 314.1 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR make: *** [cpufeatures.target.mk:121: Release/obj.target/cpufeatures/src/binding.o] Error 1
2026-06-10T05:05:57.6796327Z #16 314.1 ➤ YN0000: │ cpu-features@npm:0.0.9 STDOUT make: Leaving directory '/app/node_modules/cpu-features/build'
2026-06-10T05:05:57.6797348Z #16 314.1 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR gyp ERR! build error 
2026-06-10T05:05:57.6798241Z #16 314.1 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR gyp ERR! stack Error: `make` failed with exit code: 2
2026-06-10T05:05:57.6799401Z #16 314.1 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR gyp ERR! stack     at ChildProcess.onExit (/app/node_modules/node-gyp/lib/build.js:203:23)
2026-06-10T05:05:57.6800508Z #16 314.1 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR gyp ERR! stack     at ChildProcess.emit (node:events:509:28)
2026-06-10T05:05:57.6801640Z #16 314.1 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR gyp ERR! stack     at ChildProcess._handle.onexit (node:internal/child_process:295:12)
2026-06-10T05:05:57.6803202Z #16 314.1 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR gyp ERR! System Linux 6.14.0-1014-azure
2026-06-10T05:05:57.6804508Z #16 314.1 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR gyp ERR! command "/usr/local/bin/node" "/app/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
2026-06-10T05:05:57.6805866Z #16 314.1 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR gyp ERR! cwd /app/node_modules/cpu-features
2026-06-10T05:05:57.6806635Z #16 314.1 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR gyp ERR! node -v v24.16.0
2026-06-10T05:05:57.6807452Z #16 314.1 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR gyp ERR! node-gyp -v v9.4.0
2026-06-10T05:05:57.6808332Z #16 314.1 ➤ YN0000: │ cpu-features@npm:0.0.9 STDERR gyp ERR! not ok 
2026-06-10T05:05:57.6809353Z #16 314.2 ➤ YN0009: │ cpu-features@npm:0.0.9 couldn't be built successfully (exit code 1, logs can be found here: /tmp/xfs-f505dc4c/build.log)
2026-06-10T05:05:57.6810321Z #16 314.2 ➤ YN0007: │ ssh2@npm:1.15.0 must be built because it never has been before or the last one failed
2026-06-10T05:05:58.7027743Z #16 315.3 ➤ YN0000: │ ssh2@npm:1.15.0 STDOUT Usage Error: Couldn't find a script name "node-gyp" in the top-level (used by ssh2@npm:1.15.0). This typically happens because some package depends on "node-gyp" to build itself, but didn't list it in their dependencies. To fix that, please run "yarn add node-gyp" into your top-level workspace. You also can open an issue on the repository of the specified package to suggest them to use an optional peer dependency.
2026-06-10T05:05:58.8810935Z #16 315.3 ➤ YN0000: │ ssh2@npm:1.15.0 STDOUT 
2026-06-10T05:05:58.8813730Z #16 315.3 ➤ YN0000: │ ssh2@npm:1.15.0 STDOUT $ yarn run [--inspect] [--inspect-brk] [-T,--top-level] [-B,--binaries-only] [--require #0] <scriptName> ...
2026-06-10T05:05:58.8819964Z #16 315.4 ➤ YN0000: │ ssh2@npm:1.15.0 STDOUT Failed to build optional crypto binding
2026-06-10T05:05:58.8820801Z #16 315.4 ➤ YN0000: └ Completed in 4m 38s
2026-06-10T05:05:58.9030065Z #16 315.5 ➤ YN0000: · Done with warnings in 5m 14s
```

</details>

Still not sure why it was only a problem for one of our TechDocs sites. But we changed our Docker image to node 22 and now it works again:
```diff
- FROM node:24-trixie-slim AS base
+ FROM node:22-bookworm-slim AS base
```

I'm now mainly interested as to why the process got stuck without any error message. This would help others troubleshoot similar issues a lot faster in the future.

--- a/.changeset/fix-docker-node-version-alignment-docs.md
+++ b/.changeset/fix-docker-node-version-alignment-docs.md
@@ -2,4 +2,4 @@
 '@backstage/create-app': patch
 ---
 
-Added a comment to the example backend Dockerfiles clarifying that the host build steps (`yarn install`, `yarn build:backend`) must use the same Node version as the Docker base image (Node 24). Using a mismatched version — especially combined with a shared BuildKit Yarn cache — can cause native modules to fail silently at runtime.
+Clarified that the host build steps must use the same Node version as the Docker base image in the Dockerfile.

--- a/.changeset/fix-docker-node-version-alignment-docs.md
+++ b/.changeset/fix-docker-node-version-alignment-docs.md
@@ -1,0 +1,5 @@
+---
+'@backstage/create-app': patch
+---
+
+Added a comment to the example backend Dockerfiles clarifying that the host build steps (`yarn install`, `yarn build:backend`) must use the same Node version as the Docker base image (Node 24). Using a mismatched version — especially combined with a shared BuildKit Yarn cache — can cause native modules to fail silently at runtime.

--- a/docs/deployment/docker.md
+++ b/docs/deployment/docker.md
@@ -37,6 +37,15 @@ The required steps in the host build are to install dependencies with
 `yarn install`, generate type definitions using `yarn tsc`, and build the backend
 package with `yarn build:backend`.
 
+:::note
+Run these commands with the **same Node version as the Docker base image** (currently Node 24).
+Using a different version — for example Node 22 — causes native modules such as `better-sqlite3`
+and `cpu-features` to be compiled against a different ABI. At runtime inside the container they
+will fail to load, which can manifest as silent hangs or cryptic errors. This risk is especially
+pronounced when a BuildKit `--mount=type=cache` is used for the Yarn cache, because stale
+pre-compiled binaries from a previous Node version may be served from cache without recompilation.
+:::
+
 In a CI workflow it might look something like this, from the root:
 
 ```bash

--- a/docs/deployment/docker.md
+++ b/docs/deployment/docker.md
@@ -38,12 +38,7 @@ The required steps in the host build are to install dependencies with
 package with `yarn build:backend`.
 
 :::note
-Run these commands with the **same Node version as the Docker base image** (currently Node 24).
-Using a different version — for example Node 22 — causes native modules such as `better-sqlite3`
-and `cpu-features` to be compiled against a different ABI. At runtime inside the container they
-will fail to load, which can manifest as silent hangs or cryptic errors. This risk is especially
-pronounced when a BuildKit `--mount=type=cache` is used for the Yarn cache, because stale
-pre-compiled binaries from a previous Node version may be served from cache without recompilation.
+Run these commands with the same Node version as the Docker base image (currently Node 24). Using a different version will cause native modules to fail at runtime.
 :::
 
 In a CI workflow it might look something like this, from the root:

--- a/docs/deployment/docker.md
+++ b/docs/deployment/docker.md
@@ -38,7 +38,7 @@ The required steps in the host build are to install dependencies with
 package with `yarn build:backend`.
 
 :::note
-Run these commands with the same Node version as the Docker base image (currently Node 24). Using a different version will cause native modules to fail at runtime.
+Run these commands with the same Node version as the Docker base image. Using a different version will cause native modules to fail at runtime.
 :::
 
 In a CI workflow it might look something like this, from the root:

--- a/packages/backend/Dockerfile
+++ b/packages/backend/Dockerfile
@@ -7,11 +7,8 @@
 # yarn tsc
 # yarn build:backend
 #
-# These commands must be run with Node 24, matching this image's runtime.
-# Using a different Node version can cause native modules (e.g. better-sqlite3,
-# cpu-features) to be compiled against a different ABI, resulting in silent
-# failures or hangs at runtime — especially if Yarn's build cache is shared
-# across Node versions via a BuildKit --mount=type=cache.
+# Host build steps (yarn install, yarn build:backend) must use the same
+# Node version as the FROM image below. Mismatched versions break native modules.
 #
 # Once the commands have been run, you can build the image using `yarn build-image`
 

--- a/packages/backend/Dockerfile
+++ b/packages/backend/Dockerfile
@@ -7,6 +7,12 @@
 # yarn tsc
 # yarn build:backend
 #
+# These commands must be run with Node 24, matching this image's runtime.
+# Using a different Node version can cause native modules (e.g. better-sqlite3,
+# cpu-features) to be compiled against a different ABI, resulting in silent
+# failures or hangs at runtime — especially if Yarn's build cache is shared
+# across Node versions via a BuildKit --mount=type=cache.
+#
 # Once the commands have been run, you can build the image using `yarn build-image`
 
 FROM node:24-trixie-slim

--- a/packages/create-app/templates/default-app/packages/backend/Dockerfile
+++ b/packages/create-app/templates/default-app/packages/backend/Dockerfile
@@ -7,11 +7,8 @@
 # yarn tsc
 # yarn build:backend
 #
-# These commands must be run with Node 24, matching this image's runtime.
-# Using a different Node version can cause native modules (e.g. better-sqlite3,
-# cpu-features) to be compiled against a different ABI, resulting in silent
-# failures or hangs at runtime — especially if Yarn's build cache is shared
-# across Node versions via a BuildKit --mount=type=cache.
+# Host build steps (yarn install, yarn build:backend) must use the same
+# Node version as the FROM image below. Mismatched versions break native modules.
 #
 # Once the commands have been run, you can build the image using `yarn build-image`
 #

--- a/packages/create-app/templates/default-app/packages/backend/Dockerfile
+++ b/packages/create-app/templates/default-app/packages/backend/Dockerfile
@@ -7,6 +7,12 @@
 # yarn tsc
 # yarn build:backend
 #
+# These commands must be run with Node 24, matching this image's runtime.
+# Using a different Node version can cause native modules (e.g. better-sqlite3,
+# cpu-features) to be compiled against a different ABI, resulting in silent
+# failures or hangs at runtime — especially if Yarn's build cache is shared
+# across Node versions via a BuildKit --mount=type=cache.
+#
 # Once the commands have been run, you can build the image using `yarn build-image`
 #
 # Alternatively, there is also a multi-stage Dockerfile documented here:

--- a/packages/create-app/templates/legacy-app/packages/backend/Dockerfile
+++ b/packages/create-app/templates/legacy-app/packages/backend/Dockerfile
@@ -7,11 +7,8 @@
 # yarn tsc
 # yarn build:backend
 #
-# These commands must be run with Node 24, matching this image's runtime.
-# Using a different Node version can cause native modules (e.g. better-sqlite3,
-# cpu-features) to be compiled against a different ABI, resulting in silent
-# failures or hangs at runtime — especially if Yarn's build cache is shared
-# across Node versions via a BuildKit --mount=type=cache.
+# Host build steps (yarn install, yarn build:backend) must use the same
+# Node version as the FROM image below. Mismatched versions break native modules.
 #
 # Once the commands have been run, you can build the image using `yarn build-image`
 #

--- a/packages/create-app/templates/legacy-app/packages/backend/Dockerfile
+++ b/packages/create-app/templates/legacy-app/packages/backend/Dockerfile
@@ -7,6 +7,12 @@
 # yarn tsc
 # yarn build:backend
 #
+# These commands must be run with Node 24, matching this image's runtime.
+# Using a different Node version can cause native modules (e.g. better-sqlite3,
+# cpu-features) to be compiled against a different ABI, resulting in silent
+# failures or hangs at runtime — especially if Yarn's build cache is shared
+# across Node versions via a BuildKit --mount=type=cache.
+#
 # Once the commands have been run, you can build the image using `yarn build-image`
 #
 # Alternatively, there is also a multi-stage Dockerfile documented here:

